components/raftstore/src/store/peer_storage.rs

```diff
@@ -2197,6 +2199,7 @@ mod tests {
         }
     }
 
+    // Does not include raftdb state key.
```

Comment:
What if new states are added to raft engine? For example, we may move all states from KV engine to raft engine in the future.